### ISSUE TITLE
provider/aws: aws_vpc_peering_connection.public_internal: Error waiting for VPC Peering Connection (pcx-xxxxxxx) to become available: unexpected state 'active', wanted target 'pending-acceptance'

### ISSUE THREAD

**Issue by druidsbane** [hashicorp/terraform#9745]
### Terraform Version
Terraform v0.7.7

### Affected Resource(s)
aws_vpc_peering_connection

### Terraform Configuration Files
```
resource "aws_vpc_peering_connection" "public_internal" {
    peer_vpc_id = "${aws_vpc.public.id}"
    vpc_id = "${aws_vpc.internal.id}"
    auto_accept = true

    tags {
      Name = "VPC Peering between public and internal"
    }
}
```

### Debug Output
```
Error applying plan:

1 error(s) occurred:

* aws_vpc_peering_connection.public_internal: Error waiting for VPC Peering Connection (pcx-xxxxxx) to become available: unexpected state 'active', wanted target 'pending-acceptance'. last error: %!s(<nil>)
```

### Expected Behavior
Peering connection should have been seen as successfully created and "terraform apply" should have resumed.

### Actual Behavior
Peering connection came up fine, it was active, there was no need for "terraform apply" to fail.  Once I reran the command it was able to complete fine but the initial run failed due to the error above.

### Steps to Reproduce
1. `terraform apply`

**Comment by kwilczynski** [comment]
@druidsbane hi there! I am very sorry that you have had an issue with this resource!

I have pushed a fix which should take care of this problem!

Thank you so much for taking the time to let us know!

**Comment by druidsbane** [comment]
Thank you very much! I love Terraform!

**Comment by stack72** [comment]
Closed via #9754
